travis/travis_install_nightly

```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb # downloading updated chrome dev tools
+    apt update
```

Comment:
ADd `-q` parameter, to don't add unnecessary info to the log

Reply:
The way I added is Ok??